src/stm32f103/stm32f103x8_high.ld

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2015 Karl Palsson <karlp@tweak.net.au>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Linker script for STM32F103x8, 64k flash, 20k RAM. */
+
+/* Define memory regions. */
+/* 8k for the bootloader */
+MEMORY
+{
+	boot (rx) : ORIGIN = 0x08000000, LENGTH = 0x150
+	rom (rx)  : ORIGIN = 0x0800e600, LENGTH = 0x1A00
```

Comment:
I think it would be helpful to document that the `boot` region is used for the initial / modified vector table, since it's not entirely obvious from the name `boot` is used for that until looking at the other linker script.
I named it `bootstrap` when I did my initial branch, but since it might be more accurate to just call it `vectors` (or `vec` if you want to keep it short).

Reply:
> I named it `bootstrap` when I did my initial branch, but since it might be more accurate to just call it `vectors` (or `vec` if you want to keep it short).

I will rename it to `vectors`, seems better.